app/blueprints/assessments/templates/macros/application_overviews_table_cyp.html

```diff
@@ -0,0 +1,124 @@
+    'cohort',
+    'Search reference or organisation name',
+    assessment_statuses,
+    countries,
+    regions,
+    local_authorities,
+    round_details,
+    show_clear_filters,
+    tag_option_groups
+  ) }}
+  </form>
+</nav>
+
+{% if not application_overviews %}
+<div role="status" aria-live="assertive" class="govuk-!-margin-top-4">
+  <p class="govuk-body" autofocus>
+    <strong>No matching results.</strong>
+  </p>
+  <p class="govuk-body">Improve your results by:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>removing filters</li>
+    <li>double-checking your spelling</li>
+  </ul>
+</div>
+{% else %}
+<table class="govuk-table govuk-!-margin-top-4" id="application_overviews_table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">
+          Reference
+      </th>
+
+      {{ sortable_table_header('organisation_name', 'Organisation name', sort_order, sort_column, round_details, query_params) }}
+
+      {{ sortable_table_header('cohort', 'Cohort', sort_order, sort_column, round_details, query_params) }}
+
+      {{ sortable_table_header('funding_requested', 'Funding requested', sort_order, sort_column, round_details, query_params) }}
+
+      {{ sortable_table_header('national_or_regional', 'National or regional', sort_order, sort_column, round_details, query_params) }}
+
+      {{ sortable_table_header('tags', 'Tags', sort_order, sort_column, round_details, query_params) }}
+
+      {% if g.access_controller.has_any_assessor_role %}
+        {{ sortable_table_header('status', 'Status', sort_order, sort_column, round_details, query_params) }}
+      {% endif %}
+
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+        {% for overview in application_overviews %}
+            <tr class="govuk-table__row">
+
+                <td class="govuk-table__cell">{{ overview.short_id[-6:] }}</td>
+                <td class="govuk-table__cell"><a class="govuk-link" data-qa="organisation_name" href="{{ url_for('assessment_bp.application',application_id=overview.application_id) }}">{{ overview.organisation_name }}</a></td>
+
+                {% set cohort_display_list = [] %}
+                {% set cohort_list = overview.get("cohort", "[]") | ast_literal_eval %}
+                {% for cohrt in cohort_list %}
+                  {% set _ = cohort_display_list.append(cohort.get(cohrt))  %}
+                {% endfor %}
+
+                <td class="govuk-table__cell">{{ cohort_display_list | join(' ') }}</td>
+
+                <td class="govuk-table__cell">£{{ "{:,.2f}".format(overview.funding_amount_requested|int|round) }}</td>
+                <td class="govuk-table__cell">{{ "Regional" if overview.get("is_project_regional") else "National"}}</td>
+
+                {{ tags_table(tags[overview.application_id], tagging_purpose_config, query_params.show_tags == "ON") }}
+
+                {% if g.access_controller.has_any_assessor_role %}
```

Comment:
Since this block is repeated in all the fund-specific overviews templates, could we pull it out into it's own fragment so it's not duplicated?